docs/calicoctl/node.md

```diff
@@ -112,6 +119,10 @@ When running the `calicoctl node` command with the `--libnetwork` plugin, the
 command starts a container using the `calico/node-libnetwork` Docker image in 
 addition to starting the `calico/node` Docker image.
 
+The `--runtime` flag can be used to prevent calico from launching the
+calico-node container, instead allowing you to run the core processes
+on baremetal.
```

Comment:
Bare metal is a bit misleading.  You might use this in a VM, for example.

"... run the core Calico services via some other means. For example, you might install directly on the host and execute them via a systemd unit."